dev/README_RELEASE_PYTHON_CLIENT.md

```diff
@@ -157,7 +168,7 @@ git diff HEAD
 git checkout -b release-${VERSION}
 git add .
 git commit -m "Update Python Client to ${VERSION}${VERSION_SUFFIX}"
-git push apache release-${VERSION}
+git push origin release-${VERSION}
 ```
```

Comment:
I think meany people ehave origin set to their account so far in many of the release commands we had assumption that there is a remote pointing to Apache/airflow - maybe instead of changing we should add prerequisite to create such remote ?

Reply:
I think we have been using `origin` as remote for releases. Not sure it's written anywhere to use it but if you look at this https://github.com/apache/airflow/blob/v2-4-stable/dev/README_RELEASE_AIRFLOW.md#prepare-the-apache-airflow-package-rc from v2-4 branch before we had a release command, origin was used. Also release command default is origin. 
I think we had advocated for release clone to be different from airflow dev clone.